plugins/tableselection/plugin.js

```diff
@@ -889,6 +858,76 @@
 		setTimeout( function() {
 			editor.fire( 'afterPaste' );
 		}, 0 );
+
+		function isCustomPaste( selection, selectedCells ) {
+			var ranges = selection.getRanges(),
+				table = ranges.length && ranges[ 0 ]._getTableElement( { table: 1 } ),
+				isIgnoredTable,
+				isBoundaryWithoutTable,
+				isCollapsedPartial,
+				isTableElement;
+
+			// Do not customize paste process in following cases:
+			// 1. No cells are selected.
+			if ( !selectedCells.length ) {
+				return false;
+			}
+
+			// 2. It's a boundary position but with no table pasted.
+			isBoundaryWithoutTable = ( boundarySelection && !pastedTable );
+
+			if ( isBoundaryWithoutTable ) {
+				return false;
+			}
+
+			// 2.1 If it's boundary position and table is pasted, proceed with custom paste.
+			if ( boundarySelection ) {
```

Comment:
I did that as I didn't see a reason why pasting results should be different with and without `tableselection`. But when I think about it for the second time it's rather obvious that the results of pasting with `ignoredTableAttribute` should be the same as without `tableselection` plugin installed, and to make it work the same as with `tableselection` (as I thought) it would require a big refactor and should be done in separate issue (if it should and could be done at all - I'm not sure it's even possible), so you're right `ignoreTableAttribute` should have a higher priority here.